Getting-Started-with-Akka-http-signature.md

```diff
@@ -0,0 +1,250 @@
+
+keyRDFPointer = keyGraph.pointer
+
+keyRDFPointer: RDF#Node = 5880.026289.0-02638.0-6288.0-06411.0-23110.021178.029228.0
+```
+
+The user can transform his key graph into a one of several well-known formats before publishing it on the server. One such format is turtle. In order to do that however, more external libraries are required:
+
+```scala
+import org.w3.banana._
+import org.w3.banana.syntax._
+import ops._  
+import org.w3.banana.jena.Jena
+```
+One can then represent the rdf of the key in turtle format: 
+
+```scala
+@ val toTurtle = turtleWriter.asString(keyGraph.graph,"").get
+
+toTurtle: String = """<5880.026289.0-02638.0-6288.0-06411.0-23110.021178.029228.0>
+        <http://www.w3.org/ns/auth/cert#exponent>
+                65537 ;
+        <http://www.w3.org/ns/auth/cert#modulus>
+                "00937adccd722bc982aed4847872b81e36b890bca13166714bc2befe4d8547b6218ecd2da1eb020198a4ea00e4db6757c7dda738ec8db8b3bf211d3a3a17e196a2035bc4c79d06d8a581487d9f49e86374712b10ef500dfa242a20cab52911e2636c9d99b21fe9768ef2381989a25dc8b0b7a46531249aac27c4b8ab451a19d5fbdfa5f78b0deac9778c7ff87cf6106ae4a6433466beb21df1265bf1fc9ab9cc80d7aff8cc4d0f67ae28647e4048da8df753493b8de6a8e0961416b4b37f7012907d2e756034b8a84c6e495c8f1d81f69843ae51379571d83b38e1c08a08c3748ef75ed7ecb016d0c8426b30c8c5060a08f87f6764b0ec14667cd6f0daa1244157"^^<http://www.w3.org/2001/XMLSchema#hexBinary> .
+"""
+``` 
+This will return a String representation of the Pointed Graph in turtle format. This representation can also be saved on your local file system like so:
+
+```scala
+write(wd/"publicKey.ttl", RSAKeys.save(toTurtle))
+```
+
+As is evident, the pointer is an automatically generated Blank Node, which can be quite difficult to process. Because of this, it would be more optimal to change the pointer to not be a blank node but rather a #uri. The user can change the pointer manually by using a file editor into something simple - such as: *<#key>*
+
+### Attaching Public keys to a File/URI
+
+One can use the `cp` or the `mv` Ammonite commands to move the public key file into the test_www directory which resides within the rww-play directory. The process of attaching the file, containing the key to a URI is very similar.
+
+```scala
+cp(wd/"publicKey.ttl" , wd/"rww-play"/"test_www"/"pubKey.ttl")
+```
+
+The Rww-play library, which the Akka-http-signature library makes use of, uses symbolic links to indicate default representations of different documents. The Http content negotiation functionality is thus preserved. Http Content negotiation is a mechanism that is used for representing different representations of a resource at the same URI. The user can then specify which is best suited for them. This is quite a complex mechanism and the setup for it can vary greatly between different servers.
```

Comment:
1. rww-play is not a library but a *web server*. It should be turned into a few light weight libs but that is a job still to do. 
2. the effects of content negotiation are preserved because one can link to files such as `<card#me>` and not be forced to link to `<card.rdf#me>` or `<card.ttl#me>` which would tie the name of the object described by `#me` in the pointed graph of the document to a particular serialisation format. Read the https://www.w3.org/TR/swbp-vocab-pub/ and see if you can use the above and explain this in a paragraph. The point of having the symbolic link is so that someone using an editor such as `vi`, `emacs`, or `pico` with a mode that could follow links, would be able to follow them on the file system too...